functests/2_performance_update/updating_profile.go

```diff
@@ -64,30 +66,56 @@ var _ = Describe("[rfe_id:28761][performance] Updating parameters in performance
 			isolatedCPUSet, err := cpuset.Parse(string(*profile.Spec.CPU.Isolated))
 			Expect(err).ToNot(HaveOccurred())
 
-			var expectedBannedCPUs cpuset.CPUSet
-			if irqLoadBalancingDisabled {
-				expectedBannedCPUs = isolatedCPUSet
-			} else {
-				expectedBannedCPUs = cpuset.NewCPUSet()
-			}
+			verifyNodes := func() error {
+				var expectedBannedCPUs cpuset.CPUSet
+				if irqLoadBalancingDisabled {
+					expectedBannedCPUs = isolatedCPUSet
+				} else {
+					expectedBannedCPUs = cpuset.NewCPUSet()
+				}
 
-			for _, node := range workerRTNodes {
-				bannedCPUs, err := nodes.BannedCPUs(node)
-				Expect(err).ToNot(HaveOccurred(), fmt.Errorf("failed to extract the banned CPUs from node %s: %v", node.Name, err))
-				Expect(bannedCPUs.Equals(expectedBannedCPUs)).To(BeTrue(),
-					fmt.Errorf("banned CPUs %v do not match the expected mask %v on node %s",
-						bannedCPUs, expectedBannedCPUs, node.Name))
+				for _, node := range workerRTNodes {
+					bannedCPUs, err := nodes.BannedCPUs(node)
+					Expect(err).ToNot(HaveOccurred(), fmt.Sprintf("failed to extract the banned CPUs from node %s", node.Name))
+
+					if !bannedCPUs.Equals(expectedBannedCPUs) {
+						return fmt.Errorf("banned CPUs %v do not match the expected mask %v on node %s",
+							bannedCPUs, expectedBannedCPUs, node.Name)
+					}
+					smpAffinity, err := nodes.ExecCommandOnNode(chkDefaultSMPAffinity, &node)
```

Comment:
let's use the full name (`checkDefaultSMPAffinity`) next time

Reply:
ack